index.js

```diff
@@ -1314,6 +1314,8 @@ function Runtime() {
         });
 
         return function (target, data) {
+            if (target.isNull()) 
+                throw new Error("The target protocol is null");
```

Comment:
This conflicts with the line below – you can either pass in a `NativePointer` or an `ObjC.Object`.